Library/Formula/pngquant.rb

```diff
@@ -14,12 +14,35 @@ class Pngquant < Formula
 
   depends_on "pkg-config" => :build
   depends_on "libpng"
+  depends_on "little-cms2" => :optional
+
+  option "with-openmp", "Enable OpenMP"
+  if build.with? "openmp"
```

Comment:
We have sugar for this: `needs :openmp`

This could be better represented as `needs :openmp if build.with? "openmp"`